File: cs50/pset6/mario.py
# Подключили функцию get_int
from cs50 import get_int
# Запросили число от 1 до 8
while True:
    height = get_int("Height: ")
    if height < 9 and height > 0:
        break
# Напечатали 2 пирамиды
for i in range(height):
    for j in range(height-i-1):
        print(" ", end="")
    for j in range(i+1):
        print("#", end="")
    print(" " * 2, end="")
    for j in range(i+1):
        print("#", end="")
    print()

File: cs50/pset1/mario.c
/****************************************************************************
 
* mario.c
*
 
* Provotorov Daniil
* provotorov.d.i@gmail.com
*
 
* Draws a hill from Mario
*
 
***************************************************************************/

#include <stdio.h>
#include <cs50.h>

int main (void)
{
    int Height = 0;
    do
    {
        printf("Enter positive height from 1 to 23:");
        Height = get_int("");
    }
    while (Height < 0 || Height > 23);

    for (int i = 0; i < Height; i++)
    {
        for (int j = 0; j < Height-i-1; j++)
        {
            printf(" ");
        }

        for (int k = 0; k < i+1; k++)
        {
            printf("#");
        }
        printf("#\n");
    }

}